source/docs/software/advanced-gradlerio/profiling-with-visualvm.rst

```diff
@@ -0,0 +1,176 @@
+Profiling with VisualVM
+=======================
+
+This document is intended to familiarize the reader with the diagnostic tool that is `VisualVM <https://visualvm.github.io/>`__. VisualVM is a tool for profiling JVM based applications, such as viewing why an application is using a large amount of memory. This document assumes the reader is familiar with the *risks* associated with modifying their robot `build.gradle`. This tutorial also assumes that the user knows basic terminal/commandline knowledge.
+
+Unpacking VisualVM
+------------------
+
+To begin, `download VisualVM <https://visualvm.github.io/download.html>`__ and unpack it to the WPILib installation folder. The folder is located at ``~/wpilib/`` where ``~`` indicates the users home directory. On Windows, this is ``C:\\users\\public\\wpilib``.
```

Comment:
Is there a reason for the double slashes?

Reply:
Casing is also wrong on that path.